views.py

```
@@ -0,0 +1,483 @@
+                            # check for digit
+                        if not any(char.isdigit() for char in new_password_confirm):
+                            return HttpResponse(('Password must contain at least 1 digit.'))
+
+                            # check for letter
+                        if not any(char.isalpha() for char in new_password_confirm):
+                            return HttpResponse(('Password must contain at least 1 letter.'))
+
+                        regex = re.compile('[@_!#$%^&*()<>?/\|}{~:]')
+
+
+                        if (regex.search(new_password_confirm) == None):
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)